conf/modules.config

```diff
@@ -99,6 +99,35 @@ process {
         ]
     }
 
+    withName: 'KRAKEN2_DOWNLOAD_DB' {
+        storeDir = { "${params.kraken2_store_dir ?: params.outdir}/kraken2_databases" }
```

Comment:
Won't `params.kraken2_store_dir` always be true since it has a default? I.e. you need to specifically set this in your parameter file (or command line) to something false. Is that really what you want?